ext/telemetry/telemetry.ts

```diff
@@ -1102,6 +1107,19 @@ export function bootstrap(
     default:
       break;
   }
+
+  if (TRACING_ENABLED || METRICS_ENABLED) {
```

Comment:
if these are turned off, does this turn all otel usage into noops, or does it raise an error about them being missing?

Reply:
Noops